From 5637f2a1e9145391fcf413770c74bd16b94c4c82 Mon Sep 17 00:00:00 2001
From: Mingliang Liu <liuml07@apache.org>
Date: Mon, 29 Aug 2016 10:22:55 -0700
Subject: [PATCH 2082/2848]  HDFS-10807. Doc about upgrading to a version of
 HDFS with snapshots may be confusing.
 (Contributed by Mingliang Liu)

(cherry picked from commit 3112a2447fc81bffac909c09d59261641cd4a9ab)

Change-Id: I51baad1fd522119562bd023a7df8cdc470572ab8
---
 .../hadoop-hdfs/src/site/xdoc/HdfsSnapshots.xml    |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/site/xdoc/HdfsSnapshots.xml b/hadoop-hdfs-project/hadoop-hdfs/src/site/xdoc/HdfsSnapshots.xml
index eba1d80..57e15e5 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/site/xdoc/HdfsSnapshots.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/site/xdoc/HdfsSnapshots.xml
@@ -109,7 +109,7 @@
   <p>
     The HDFS snapshot feature introduces a new reserved path name used to
     interact with snapshots: <tt>.snapshot</tt>. When upgrading from an
-    older version of HDFS, existing paths named <tt>.snapshot</tt> need
+    older version of HDFS which does not support snapshots, existing paths named <tt>.snapshot</tt> need
     to first be renamed or deleted to avoid conflicting with the reserved path.
     See the upgrade section in
     <a href="HdfsUserGuide.html#Upgrade_and_Rollback">the HDFS user guide</a>
-- 
1.7.9.5

